### ISSUE TITLE
Add option to report the metrics for the target and analog volcanoes

### ISSUE THREAD

**Issue by maxrjones** [BritishGeologicalSurvey/pyvolcans#16]
This feature request is to allow the user to query the variable values for each category for the target and/or analog volcanoes in addition to the analogy metrics. For example, a flag that would report `Ts: 0.1250; Ip - Oceanic crust` etc. for Kilauea).

**Comment by PTierz** [comment]
Dear @meghanrjones,

I sincerely hope that this message finds you well.

I just wanted to let you know that we have preliminarily addressed Issue #16 in a new branch called [`volcano-id-profile`](https://github.com/BritishGeologicalSurvey/pyvolcans/tree/volcano-id-profile).

We still need to add a few improvements before we merge the branch into [`main`](https://github.com/BritishGeologicalSurvey/pyvolcans/tree/main) but please feel free to explore the new functionalities of the code and let us know if they appear to be satisfactory as regards your request.

We will aim at completing the merge by 15th November 2021.

Very many thanks in advance,
Pablo

**Comment by volcan01010** [comment]
To pretty print a dictionary:

```python
import json

my_volcano = {'name': 'Hekla', 'other_attribute': 1}
print(json.dumps(my_volcano, indent=2, sort_keys=False))
```

`sort_keys` will keep them in the order that they were entered.

**Comment by PTierz** [comment]
Dear @meghanrjones,

I hope this message finds you well.

First of all, please let us thank you for your review of our `PyVOLCANS` manuscript and for the very useful suggestions, including raising this issue, which have helped us to significantly improve the software.

Regarding the current issue, please see the changes in the now merged `volcano-id-profile` branch (#17).

In the current version of the manuscript, running `pyvolcans Kilauea -v` should provide not only the Top 10 analogues to Kīlauea (using a default, equal-weighting scheme) but also the _ID profile_ for the target volcano, as stored in data from GVP 4.6.7 and the merged volcano morphology database used in [Tierz et al. (2019)](https://doi.org/10.1007/s00445-019-1336-3).

Finally, running `pyvolcans Kilauea -v --output_volcano_data` should additionally save a `.json` file containing Kīlauea's  _ID profile_, and running `pyvolcans Kilauea -v --output_analogues_data` should save a different `.json` file containing the _ID profile_ for all the Top 10 analogues to Kīlauea, the latter considering the default, equal-weighting scheme used.

We sincerely hope that these changes will satisfy your request. If this is the case, please feel free to close the issue yourself, or let me know that the changes look appropriate, so I can close the issue.

Thank you very much in advance for your support.

Best regards,
Pablo

**Comment by maxrjones** [comment]
Thanks @PTierz. I will close this issue now and review the changes using the checklist in the joss-reviews issue.
